components/consensusmanager/src/session.rs

```diff
@@ -23,9 +23,9 @@ pub use tokio::task::spawn_blocking;
 use crate::BlockProcessingBatch;
 
 #[derive(Clone)]
-pub struct SessionOwnedReadGuard(Arc<RfRwLockOwnedReadGuard>);
+pub struct SessionOwnedReadGuard(pub Arc<RfRwLockOwnedReadGuard>);
 
-pub struct SessionReadGuard<'a>(RfRwLockReadGuard<'a>);
+pub struct SessionReadGuard<'a>(pub RfRwLockReadGuard<'a>);
```

Comment:
Did this change occur  due to inter?
If so I would suggest to suppress it instead of making the field public

Reply:
changed: https://github.com/kaspanet/rusty-kaspa/pull/442/commits/319a6247dda2546baf62fc4f8ed6aac705029fc1